lib/galaxy/webapps/galaxy/api/datasets.py

```diff
@@ -248,7 +249,13 @@ def display(
     ):
         """Streams the preview contents of a dataset to be displayed in a browser."""
         extra_params = get_query_parameters_from_request_excluding(request, {"preview", "filename", "to_ext", "raw"})
-        display_data, headers = self.service.display(trans, history_content_id, history_id, preview, filename, to_ext, raw, **extra_params)
+        display_data, headers = self.service.display(
+            trans, history_content_id, preview, filename, to_ext, raw, **extra_params
+        )
```

Comment:
```suggestion
        display_data, headers = self.service.display(
            trans, history_content_id, history_id, preview, filename, to_ext, raw, **extra_params
        )
```
Was dropping `history_id` intentional?

Reply:
Nope, that came from a big WIP branch